scripts/retract_registrations.py

```diff
@@ -0,0 +1,56 @@
+"""Script for retracting pending retractions that are more than 48 hours old."""
+
+import datetime
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.auth import Auth
+from website import models, settings
+from website.app import init_app
+from website.project.model import NodeLog
+from scripts import utils as scripts_utils
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def main(dry_run=True):
+    pending_retractions = models.Retraction.find(Q('state', 'eq', models.Retraction.PENDING))
+    for retraction in pending_retractions:
+        if should_be_retracted(retraction):
+            if dry_run:
+                logger.warn('Dry run mode')
+            parent_registration = models.Node.find_one(Q('retraction', 'eq', retraction))
+            logger.warn(
+                'Retraction {0} approved. Retracting registration {1}'
+                .format(retraction._id, parent_registration._id)
+            )
+            if not dry_run:
+                retraction.state = models.Retraction.RETRACTED
```

Comment:
As per previous comment, use a transaction here.

There seems to be a lot of shared code between `embargo_registrations.py` and `retraction_registrations.py`? Can you DRY this up?